docs/detections/rules-ui-create.asciidoc

```diff
@@ -255,6 +266,21 @@ field values.
 +
 . Continue with <<preview-rules, previewing the rule>> (optional) or click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
 
+[discrete]
+[[create-new-terms-rule]]
+==== Create a new terms rule
+
+. To create a rule that searches for each new term detected in source documents, select *New Terms*, then:
+.. Specify what data to search by entering individual {es} index patterns or selecting an existing data view.
+.. Use the filter and query fields to create the criteria used for detecting
+alerts.
++
+NOTE: NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,17,17]) and queries from saved Timelines (`Import query from saved Timeline`) as rule conditions.
++
+.. Use the `Fields` menu to select a field to check for new terms.
+.. Use the `History Window Size` to specify the time range to search in minutes, hours, or days to determine if a term is new.
```

Comment:
Same thing with bold here -- these are UI elements, not entered values or code examples:
```suggestion
.. Use the *Fields* menu to select a field to check for new terms.
.. Use the *History Window Size* to specify the time range to search in minutes, hours, or days to determine if a term is new.
```